functions/create_draft/interactivity_handler.ts

```diff
@@ -65,14 +67,51 @@ export const openDraftEditView: BlockActionHandler<
     if (!viewsOpenResp.ok) {
       const draftEditModalErrorMsg =
         `Error opening up the draft edit modal view. Contact the app maintainers with the following - (Error detail: ${viewsOpenResp.error}`;
-      console.log(draftEditModalErrorMsg);
+      console.error(draftEditModalErrorMsg);
 
       await client.functions.completeError({
         function_execution_id: body.function_data.execution_id,
         error: draftEditModalErrorMsg,
       });
     }
   }
+
+  // If the user selects to discard the draft message
+  if (action.selected_option.value === "discard_message_overflow") {
+    const id = action.block_id;
+    const message_ts = body.message?.ts || "";
+
+    // Delete the draft from the 'drafts' Datstore
+    const deleteResp = await client.apps.datastore.delete({
+      datastore: DraftDatastore.name,
+      id: id,
+    });
+
+    // If the DELETE datastore record operation fails, log the error and complete the function with an error
+    if (!deleteResp.ok) {
+      const deleteDraftErrorMsg =
+        `Error deleting draft with id ${id}. Contact the app maintainers with the following - (Error detail: ${deleteResp.error})`;
+      console.error(deleteDraftErrorMsg);
+
+      await client.functions.completeError({
+        function_execution_id: body.function_data.execution_id,
+        error: deleteDraftErrorMsg,
+      });
+    }
+
+    // Delete the draft message from the Draft Channel
+    const updateResp = await client.chat.delete({
+      channel: inputs.channel,
+      ts: message_ts,
```

Comment:
We might want to skip an attempt at `chat.delete` if the `message_ts` is empty. Not sure when this can happen, but better to avoid this if it's possible IMO!

Reply:
I added a Try/Catch block around both the Delete Datastore and Delete Message options - Let me know if that's what you meant or if I should try doing something else?  Thanks much!